### PR TITLE
improve language detection

### PR DESCRIPTION
**Pull request by spowelljr**
Fixes https://github.com/kubernetes/minikube/issues/13894

We were using an old language detection package that was archived and was missing checks listed in UNIX documentation.

Updated to a different detection package that contains language detection for all OSes.

### CODE REVIEW

**Review by klaases — APPROVED** [review]
Nice! Much cleaner implementation.

File: pkg/minikube/translate/translate.go [hunk 1]
```diff
@@ -62,20 +60,11 @@ func T(s string) string {
 
 // DetermineLocale finds the system locale and sets the preferred language for output appropriately.
 func DetermineLocale() {
-	var locale string
-	// Allow windows users to overload the same env vars as unix users
-	if runtime.GOOS == "windows" {
-		locale = os.Getenv("LC_ALL")
```
Comment (medyagh) [inline pkg/minikube/translate/translate.go:68]: can u verify after this PR LC_ALL env variable can still override the language ?

Comment (spowelljr) [reply pkg/minikube/translate/translate.go:68]: The comment in the library says
`// LC_ALL will overwrite all LC_* options.`

Comment (medyagh) [reply pkg/minikube/translate/translate.go:68]: lets paste the output of LC_ALL=fr minikube profile list and verify

Comment (spowelljr) [reply pkg/minikube/translate/translate.go:68]:
```
$ echo $LC_ALL
fr
$ ./out/minikube profile list

🤹  Fermeture en raison de MK_USAGE_NO_PROFILE : Aucun profil minikube n'a été trouvé.
💡  Suggestion : 

    Vous pouvez en créer un en utilisant 'minikube start'.
```

**Pull request merged**